mobile-app/app/screens/AppNavigator/screens/Auctions/screens/PlaceBidScreen.tsx

```diff
@@ -0,0 +1,263 @@
+
+export function PlaceBidScreen (props: Props): JSX.Element {
+  const { batch, vault } = props.route.params
+  const tokens = useTokensAPI()
+  const ownedToken = tokens.find(token => token.id === batch.loan.id)
+  const { minNextBidInToken, totalLoanAmountInUSD } = useAuctionBidValue(batch, vault.liquidationPenalty, vault.loanScheme.interestRate)
+  const estimatedFees = new BigNumber(0.002)
+
+  const navigation = useNavigation<NavigationProp<AuctionsParamList>>()
+  const hasPendingJob = useSelector((state: RootState) => hasTxQueued(state.transactionQueue))
+  const hasPendingBroadcastJob = useSelector((state: RootState) => hasBroadcastQueued(state.ocean))
+  const blockCount = useSelector((state: RootState) => state.block.count) ?? 0
+
+  const [bidAmount, setBidAmount] = useState<string>('')
+
+  const onBidMinAmount = (val: string): void => {
+    const ownedTokenAmount = ownedToken === undefined ? '0' : ownedToken.amount
+    const newBidAmount = new BigNumber(ownedTokenAmount).isGreaterThan(val) ? val : ownedTokenAmount
+
+    setBidAmount(newBidAmount)
+  }
+
+  const onSubmit = async (): Promise<void> => {
+    if (hasPendingJob || hasPendingBroadcastJob) {
+      return
+    }
+
+    navigation.navigate('ConfirmPlaceBidScreen', {
+      batch,
+      bidAmount: new BigNumber(bidAmount),
+      estimatedFees,
+      totalAuctionValue: totalLoanAmountInUSD,
+      vault
+    })
+  }
+
+  const isValidMinBid = new BigNumber(bidAmount).gte(minNextBidInToken)
+
+  return (
+    <ThemedScrollView
+      testID='place_bid_screen'
+      contentContainerStyle={tailwind('flex flex-col justify-between py-6 pb-8 px-4 h-full')}
+    >
+      <View>
+        <BidSummaryCard
+          displaySymbol={batch.loan.displaySymbol}
+          collateralDisplaySymbols={batch.collaterals.map(collateral => collateral.displaySymbol)}
+          blockCount={blockCount}
+          liquidationHeight={vault.liquidationHeight}
+          minNextBid={minNextBidInToken}
+          totalAuctionValue={totalLoanAmountInUSD}
+        />
+
+        <WalletTextInput
+          autoCapitalize='none'
+          onChange={(e: NativeSyntheticEvent<TextInputChangeEventData>) => {
+            setBidAmount(e.nativeEvent.text)
+          }}
+          placeholder='0.00'
+          style={tailwind('flex-grow w-2/5')}
+          value={bidAmount}
+          displayClearButton={new BigNumber(bidAmount).gte('0.00')}
+          onClearButtonPress={() => setBidAmount('0.00')}
+          title={translate('screens/PlaceBidScreen', 'How much do you want to place?')}
+          inputType='numeric'
+          valid={isValidMinBid}
+          inlineText={{
+            type: 'error',
+            text: isValidMinBid ? undefined : translate('screens/PlaceBidScreen', 'Bid amount is lower than required')
+          }}
+        >
+          <SetAmountButton
+            amount={new BigNumber(minNextBidInToken)}
+            onPress={onBidMinAmount}
```

Comment:
I think pressing `MIN. BID` button should always input the `min. next bid` even though user doesnt have enough balance. And then we'll display the corresponding validation. What do you think?

Reply:
Come to think of it, it says `MIN. BID`, I agree. Will update